tools/docker-images/clp-package/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+repo_root=${script_dir}/../../../
+
+build_cmd=(
+    docker build
+    --tag "clp-package:dev"
```

Comment:
Sorry, in hindsight, I think we should include the architecture in the image name (to clearly indicate to users that it's not an ARM image). What do you think?

Reply:
1. while i see misusing an image built on a different arch could harm performance, 
2. using a unified name should make our container orchestration infra platform agnostic.

i feel the benefits that (2.) brings outweighs the harm (1.) could potentially cause. (1.) should not be an issue once we start looking into building mutli platform images